deploy/docs/v1_conf_examples.md

```diff
@@ -0,0 +1,130 @@
+# Configuring Fluentd
+
+Until now, Helm users have not been able to modify their Fluentd configuration outside of the specific parameters that we exposed in the `values.yaml` file. Now, we expose the ability to modify the Fluentd configuration as needed. 
+
+Some use-cases include : 
+ - custom log pipelines, 
+ - adding Fluentd filter plugins (ex: fluentd throttle plugin), or 
+ - adding Fluentd output plugins (ex: forward to both Sumo and S3)
+ - additinal configurantion for sumologic output plugin
+
+
+Below you can see a few examples of how this configuration can be set.
+
+### Custom Log Pipelines
+Now we have exposed an `extraLogs` paramter inside the `logs.containers` section of the `values.yaml` where you can add the output plugin for the custom log pipeline.
+
+```yaml
+fluentd:
+  logs:
+    extraLogs: |-
+      <match custom.tag>
+        @type sumologic
+        @id sumologic.endpoint.custom.log.endpoint
+        source_category k8s/custom
+        source_name custom
+        data_type logs
+        log_key log
+        endpoint "#{ENV['SUMO_ENDPOINT_CUSTOM_LOG']}"
+        proxy_uri "#{ENV['PROXY_URI']}"
+        verify_ssl "#{ENV['VERIFY_SSL']}"
+        log_format "json_merge"
+        add_timestamp "false"
+        <buffer>
+          @type memory
+          @include buffer.output.conf
+        </buffer>
+      </match>
+```
+
+### Adding Fluentd Filter plugin
+
+You can add any of the fluentd filter plugins in the `extraFilterPluginConf` section to filter data as per your needs.
+
+The below example uses the `grep` filter to match any event that satisfies the following conditions:
+ - The value of the "message" field contains "cool"
+ - The value of the "message" field does NOT contain "uncool"
+
+```yaml
+fluentd:
+  logs:
+    containers:
+      extraFilterPluginConf: |-
+        <filter containers.**>
+          @type grep
+          <regexp>
+            key message
+            pattern /cool/
+          </regexp>
+          <exclude>
+            key message
+            pattern /uncool/
+          </exclude> 
+        </filter>
+```
+Reference documentation: [Fluentd Filter Plugin](#https://docs.fluentd.org/filter)
+
+## Override Fluentd Output plugin to forward data to Sumo as well as S3
+The example below shows how you can override the entire output section for the container logs pipeline.
```

Comment:
Somewhere in this doc (maybe at the very top of the doc, as a note? or maybe within each of these sections? I'm not sure...) we should mention the fact that the Helm templating like
```
{{ .Values.fluentd.foo.bar | quote }}
```
will NOT work if they are specified within one of these plugin conf sections, since they are in the `values.yaml` file and are therefore interpreted as literal strings